library/core/src/option.rs

```diff
@@ -963,9 +963,11 @@ impl<T> Option<T> {
     /// assert_eq!(y, None);
     /// ```
     #[inline]
+    #[rustc_const_unstable(feature = "const_option", issue = "67441")]
     #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
The const stability attribute usually goes after the normal stability attribute.